x/concentrated-liquidity/incentives.go

```diff
@@ -28,6 +27,7 @@ func (k Keeper) createUptimeAccumulators(ctx sdk.Context, poolId uint64) error {
 	return nil
 }
 
+// TODO: write go test
```

Comment:
Is there an issue for this?

Reply:
it's not too much so i think i'll just write the test in this PR

Reply:
yessir https://github.com/osmosis-labs/osmosis/issues/5088

Reply:
Can you link this issue directly in the TODO as well please?

Reply:
@stackman27 please ping for merge once added

Reply:
test already got merged https://github.com/osmosis-labs/osmosis/pull/5090